.github/workflows/aws_tfhe_integer_tests.yml

```diff
@@ -9,52 +9,38 @@ env:
 on:
   # Allows you to run this workflow manually from the Actions tab as an alternative.
   workflow_dispatch:
-    # All the inputs are provided by Slab
-    inputs:
-      instance_id:
-        description: "AWS instance ID"
-        type: string
-      instance_image_id:
-        description: "AWS instance AMI ID"
-        type: string
-      instance_type:
-        description: "AWS instance product type"
-        type: string
-      runner_name:
-        description: "Action runner name"
-        type: string
-      request_id:
-        description: "Slab request ID"
-        type: string
-      fork_repo:
-        description: "Name of forked repo as user/repo"
-        type: string
-      fork_git_sha:
-        description: "Git SHA to checkout from fork"
-        type: string
+  pull_request_review:
+    types: [ submitted ]
 
 jobs:
-  integer-tests:
+  setup-ec2:
+    name: Setup EC2 instance (unsigned-integer-tests)
+    if: ${{ github.event_name == 'workflow_dispatch' || github.event.review.state == 'APPROVED' }}
```

Comment:
approved all caps ?

Reply:
According to Github documentation, yes but it's weird since our current implementation (lowercase) is working.

Reply:
Anyway, it still doesn't work.